model/scratch/dffml_model_scratch/model/slr.py

```diff
@@ -0,0 +1,120 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2019 Intel Corporation
+'''
+Description of what this model does
+'''
+import os
+import abc
+from typing import AsyncIterator, Tuple, Any, List, Optional, NamedTuple, Dict
+from statistics import mean
```

Comment:
Please implement the `mean` function yourself (We want to avoid importing small unmaintained libraries, numpy is fine that's well maintained).

Reply:
statistics is a part of python standard library, I don't think it will create a problem. I will replace it if you say...

Reply:
Ah yes it is my bad, I checked pypi without checking the standard library :) let's leave it then!